src/v2/cookbook/form-validation.md

```diff
@@ -0,0 +1,322 @@
+---
+title: Form Validation
+type: cookbook
+order: 1.2
+---
+
+## Simple Example
+
+Form validation is natively supported by the browser, but sometimes different browsers will handle things in a manner which makes relying on it a bit tricky. Even when validation is supported perfectly, there may be times when custom validations are needed and a more manual, Vue-based solution may be more appropriate. Let's begin with a simple example.
+
+For my first example, let's create something as simple as possible. Given a form of three fields, make two required. Let's look at the HTML first:

```

Comment:
This is a bit redundant as you said, "let's begin with a simple example" above. We try to use "we" instead of "my" in teaching as well. I'd suggest beginning the second paragraph with "Given a form..."